### DISCUSSION TITLE
Play版无法激活

### DISCUSSION THREAD

**Discussion by JackA1ltman** [Tornaco/Thanox#889] [Q&A]
我可以确认Play服务和商店正常，但是thanox无法激活，如图，而且，安装提示无法安装，但其实已经安装了
![Screenshot_20240328-113008.jpg](https://github.com/Tornaco/Thanox/assets/82563979/24bb6f20-dca9-4c3a-9a0e-52d58c4f5a2a)

![Screenshot_20240328-113248.jpg](https://github.com/Tornaco/Thanox/assets/82563979/bcde6c88-bcef-455b-a155-937c2ff17e4d)

**Comment by Tornaco** [comment]
建议检查你的谷歌服务框架，验证期间：不要用xp模块干涉谷歌服务。
